API/projectClass/parametersClass.m

```diff
@@ -1,26 +1,48 @@
 classdef parametersClass < tableUtilities
-    % This is the class definition for
-    % the parameters block.
-    
+    % ``parametersClass`` manages the parameters for the project. It provides methods to add, update and remove parameters.
+    % Each parameter is stored as a row in a table and consist of a name, a value with its minimum and maximum limits, a flag indicating 
+    % if the parameter should be fitted in the calculation, and information for Bayesian calculations such as whether the prior is uniform 
+    % or gaussian, then mu and sigma for gaussian priors. ``parametersClass`` will be initialised with a default first parameter if no 
+    % arguments are provided otherwise the provided arguments will be used to create the first parameter.  
```

Comment:
I'm wondering if this would read clearer as:

"if the parameter should be fitted in the calculation, and priors for Bayesian calculations. The type of the prior (either "uniform", "gaussian", or "jeffreys") is included alongside the mean (mu) and standard deviation (sigma) of the Gaussian distribution for Gaussian priors. ``parametersClass`` will be initialised . . ."

Note that Alex has filed the PR for the jeffreys prior, and it will probably be merged today.